components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/AbstractApplicationAuthenticator.java

```diff
@@ -179,17 +187,32 @@ protected void publishAuthenticationStepAttempt(HttpServletRequest request, Auth
             if (success) {
                 authnDataPublisherProxy.publishAuthenticationStepSuccess(request, context,
                         unmodifiableParamMap);
+                // Resetting the authenticator start time to null since the step event is Success and for the next
+                // step event start time will be added in DefaultStepHandler handle method.
```

Comment:
We can use a multiline comment here.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/3159